src/helpers/game_utils.ts

```diff
@@ -46,6 +48,15 @@ async function getFilteredSongList(guildPreference: GuildPreference, ignoredVide
     if (ignoredVideoIds && ignoredVideoIds.length > 0) {
         result = result.filter((song) => !ignoredVideoIds.includes(song.youtubeLink));
     }
+    if (guildPreference.getSQLGender() === GENDER.ALTERNATING && alternatingGender !== undefined) {
```

Comment:
Would alternatingGender ever be undefined when its in alternating mode?

Reply:
Yeah, when getSongCount calls getFilteredSongList and it's set to alternating genders